### ISSUE TITLE
Docker status showing "health: starting" continuously

### ISSUE THREAD

**Issue by akhendra** [wiktorn/Overpass-API#112]
This is continuation of the issue **https://github.com/wiktorn/Overpass-API/issues/111**
I see the **"db"** folder is created and indexed data seen in the docker container
Find the attached screenshot of inside the docker container.

But the still the docker container status is showing  "health: starting".
![image](https://github.com/wiktorn/Overpass-API/assets/150430008/b50dd542-49e0-4057-83fa-4ba9b8c36c84)

**Comment by akhendra** [comment]
The docker is still showing the "starting" only even after 18 hours
![image](https://github.com/wiktorn/Overpass-API/assets/150430008/05143728-8589-447c-9ff1-46e75670bd7c)

Any problem with the data?

**Comment by wiktorn** [comment]
If you are processing planet, it may take a few days to ingest that. Did it finish?

**Comment by akhendra** [comment]
> If you are processing planet, it may take a few days to ingest that. Did it finish?

It is taking more time to ingest the data as I did this for asia planet. I checked the system configuration and it has more than 1GB RAM left but 2 cores. So I decided to go for higher system configuration which has 32GB RAM and 84cores. 
I deleted the docker instance and ran again and it took almost two days to compute with that powerful machine.
Now it is working fine.
Thank you wiktron.
